src/openforms/analytics_tools/templates/analytics_tools/siteimprove.html

```diff
@@ -9,8 +9,9 @@
 <script id="siteimprove-analytics" type="text/javascript" nonce="{{ request.csp_nonce }}">
 /*<![CDATA[*/
 (function() {
-var sz = document.createElement('script'); sz.type = 'text/javascript'; sz.async = true;
+var sz = document.createElement('script'); sz.type = 'text/javascript';  sz.async = true;
 sz.src = '//siteimproveanalytics.com/js/siteanalyze_{{ analytics_tools_config.siteimprove_id|escapejs }}.js';
+sz.nonce="{{ request.csp_nonce }}";
```

Comment:
```suggestion
var sz = document.createElement('script'); sz.type = 'text/javascript'; sz.async = true;
sz.src = '//siteimproveanalytics.com/js/siteanalyze_{{ analytics_tools_config.siteimprove_id|escapejs }}.js';
sz.nonce = '{{ request.csp_nonce }}';
```

Reply:
I thought this was only required for the Siteimprove script because it creates another script on the fly. That's not the case with the other analytics tools. The `nonce` is already specified on them

Reply:
you are right - it looks like this is the only one creating a script tag on the fly.

Reply:
as discussed in the office - this indirection to create the script tag dynamically seems excessive - can we try replacing this with:

```django
<script type="text/javascript" src="//siteimproveanalytics.com/js/siteanalyze_{{ analytics_tools_config.siteimprove_id|escapejs }}.js" async></script>
```